android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -2444,6 +2446,34 @@ public void showIntroductionScreenForDeeplink(@NonNull String deepLink,
     IntroductionDialogFragment.show(getSupportFragmentManager(), deepLink, factory);
   }
 
+  @Override
+  public boolean onKeyUp(int keyCode, KeyEvent event)
+  {
+    switch (keyCode)
+    {
+      case KeyEvent.KEYCODE_DPAD_DOWN:
+        Statistics.INSTANCE.trackEvent(Statistics.EventName.ZOOM_OUT);
+        MapFragment.nativeScaleMinus();
+        return true;
+      case KeyEvent.KEYCODE_DPAD_UP:
+        Statistics.INSTANCE.trackEvent(Statistics.EventName.ZOOM_IN);
+        MapFragment.nativeScalePlus();
+        return true;
+      case KeyEvent.KEYCODE_ESCAPE:
+        if (MwmApplication.get().backUrl() != null)
```

Comment:
Check a string with TextUtils.isEmpty method please. Since the string may be empty.